### DISCUSSION TITLE
Itens superiores <Possivel Bug>

### DISCUSSION THREAD

**Discussion by Wsneto23** [YuriAlessandro/gerador-ficha-tormenta20#369] [Problemas]
Opa, Sempre que gero uma arma superior a primeira geração de arma sempre só vem com 1 atributo extra, nunca 2 ou mais

Fiz teste com Espada Bastarda umas 10x e sempre foi somente 1 atributo

**Comment by YuriAlessandro** [comment]
Eu testei aqui e consegui alguns casos onde a primeira geração veio com até 3 modificações. Na verdade em alguns minutos apareceram vários casos desse. Será que não foi só azar mesmo?

**Reply by Wsneto23** [reply]
Posso ter sido MUITO AZARADO hahaha

eu fiz mais de 10 testes, inclusive comecei a considerar sempre a partir do segundo item

Mas, real pode ter sido um azar gigante, vou usar mais vezes e volto
